### PR TITLE
Adding kernbench test to avocado

### PR DESCRIPTION
**Pull request by sudeeshjohn**
Kernbench is a standard CPU benchmark. Runs a kernel
compile and measures the performance by using time
command. By default it runs only once with 2 * number
of CPUs.

Signed-off-by: Sudeesh John <sudeesh@linux.vnet.ibm.com>

### CODE REVIEW

File: perf/kernbench.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        timefile = ("%s/time_file" % kern.build_dir)
+        # Build kernel
+        user_time = 0
+        system_time = 0
+        elapsed_time = 0
+        for run in range(iterations):
+            log.info("Iteration: %s" % run)
+            self.time_build(kern, threads, timefile, "")
+            # Processing the timefile
+            results = open(timefile).readline()
```
Comment (PraveenPenguin) [inline perf/kernbench.py]: can `results = open(timefile).readline()` be` results = open(timefile).readline().strip()`  ?

File: perf/kernbench.py.data/config [hunk 2]
```diff
@@ -0,0 +1,2980 @@
+# Kernel Configuration
```
Comment (PraveenPenguin) [inline perf/kernbench.py.data/config]: can we use running kernel config file which reside /boot/config* 

python code can be 
```
import platform
-------------
kernel_version = platform.uname()[2]
config_file = '/boot/config-' + kernel_version
-------------- 
```

File: perf/kernbench.py (hunk 1 above)
Comment (sudeeshjohn) [inline perf/kernbench.py]: yea I can do that.

File: perf/kernbench.py.data/config (hunk 2 above)
Comment (sudeeshjohn) [inline perf/kernbench.py.data/config]: @PraveenPenguin , I'll do that..I was just following the autotest method of kernbench. thanks for your comment.

Comment (sudeeshjohn) [inline perf/kernbench.py.data/config]: closing this request as there is a new one opens #35 with the changes.

**Pull request closed without merging**